### DISCUSSION TITLE
Configuration Interface Examples

### DISCUSSION THREAD

**Discussion by garroble** [semuconsulting/pyubx2#146] [General]
I believe there is an erratum on the Configuration Interface Examples where it says:
cfgData = [("CFG_UART1_BAUDRATE", 9600), (0x40530001, 115200)]
And the correct key should be
cfgData = [("CFG_**UART1**_BAUDRATE", 9600), (0x405**2**0001, 115200)]
Best regards.

**Comment by semuadmin** [comment]
should be fixed now
